data/pages-content/thankyou.md

```diff
@@ -1,368 +1,378 @@
 
+### Gonzo - AV Team
 Big thanks to our AV team [Gonzo Design](https://www.gonzodesign.it/) who have
 been streaming, recording & editing our talks, and more importantly,
 experimenting with us since 2017! Here are the folks who helped us with
 streaming and the smooth hybrid experience:
-
 - Matteo Vanni
 - Consuelo Cipriani
 - Vincenzo Cicanese
 - Simone Giorgi
 - Domenico Giovane
 - Ambra Lorenzetti
 - Alessandro Arese Visconti
-- Valentina Bucca
-
-### Conference Logo
-
-Our logo was again designed by Jessica Peña Moro from
-[Simetriko Diseño y Publicidad](https://simetriko.com). Jessica has been our
-logo designer since 2015.
-
-### Conference Website
-
-Our website was newly designed this year by
-[The Developer Society](https://www.dev.ngo/) and deployed to
-[Vercel](https://vercel.com/).
-
-You can find the source code of our website on
-[GitHub](https://github.com/EuroPython/website)
-
-### Conference Photographers
-
-The official [conference photos](https://www.flickr.com/groups/14792291@N24/)
-were taken by:
+<!-- - FIXME -->
 
-- [Larissa Oliveira](https://linktr.ee/larissaoliveiraphotography) during the
-  Tutorial & Conference days; and by
-- [Liam McCafferty](https://mccafferty.eu/) during the Sprint Weekend.
 
-### The Full Shilling - Social Event Performers
+### Conference Photographer
 
-The social event was made ever so special by our Dublin collective of musicians
-“The Full Shilling":
+The official EuroPython conference and social event photos (to be released) were taken by:
+- [ Braulio Lara](https://www.brauliolaraph.com/)
 
-- Claire Sherry (banjo)
-- Eoghan Scott (guitar/vocals)
-- Sarah-May Rogers (vocals/fiddle),
-- Ciaran McManus (dancer)
-- Jean Kennedy (dancer)
+### Hazafele - Social Event Band
 
-### Special mentions
+The social event was made ever so special by the band "[Hazafele](https://www.facebook.com/profile.php?id=100053175145125)"
+- Šimon Jakubíček (double bass)
+- David Golas (guitar)
+- Michal Grossmann (violin)
+- Marek Šnorich (accordion)
 
-This year we migrated to a bunch of new tools, and stuck with some old ones. We
-would like to thank their developers and the companies behind them.
+### Special Mentions
 
-The conference wouldn't be possible without:
+We would like to thank the developers and companies behind the tools we use. The conference wouldn't be possible without:
 
-- [Pretix](https://pretix.eu/about/en/) & [venueless](https://venueless.org/en/)
-  – special thanks to Dennis Lichtenthäler & Raphael Michel
+- [Pretix](https://pretix.eu/about/en/)
+  – special thanks to Raphael Michel & Martin Gross
 - [Pretalx](https://pretalx.com/p/about/) – special thanks to Tobias Kunze
 - [1Password](https://1password.com/)
 - [Zammad](https://zammad.com/en) (self-hosted)
```

Comment:
I think, (self-hosted) can be omitted

Reply:
Thank you for the review @Kislovskiy! Addressed in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :blush: